src/app/components/card/card.component.html

```diff
@@ -48,9 +48,7 @@
         tooltip="unassigned"
         placement="top"></span>
     </div>
-    <div class="margin-right-15 padding-right-15" tooltip="{{cardValue.title}}" placement="top">
-      {{ cardValue.title | truncate : 80 }}
-    </div>
+    <div class="margin-right-15 padding-right-15" tooltip="{{cardValue.title}}" placement="top" [innerHTML]="cardValue.title | truncate : 80"></div>
```

Comment:
Can you maintain the indentation and line break? something like this - 
```
<div class="margin-right-15 padding-right-15" 
        tooltip="{{cardValue.title}}" 
        placement="top" 
        [innerHTML]="cardValue.title | truncate : 80">
</div>
```

Reply:
Done.